lib/ansible/modules/monitoring/circonus_annotation.py

```diff
@@ -91,7 +89,11 @@
 import json
 import time
 
-import requests
+try:
+    import requests
+    HAS_REQUESTS = True
```

Comment:
I made some tests:

* using Python 3: requests `2.0.0` is required, there is an error with both `1.0.0` and `1.2.3`:
* using Python 2, the module works fine using `1.0.0`, `2.0.0`, `2.18.13`, there is an error with `0.14.2` (at this time, `response.json` [wasn't](https://media.readthedocs.org/pdf/requests/v0.14.2/requests.pdf) a method).

Error using Python 3 and requests < 2.0.0:

```
lib/ansible/modules/monitoring/circonus_annotation.py:155: in main
    resp = post_annotation(annotation, module.params['api_key'])
lib/ansible/modules/monitoring/circonus_annotation.py:106: in post_annotation
    headers=build_headers(api_key), data=json.dumps(annotation))
../../../.venv/ansible3_tests/lib/python3.5/site-packages/requests/api.py:88: in post
    return request('post', url, data=data, **kwargs)
../../../.venv/ansible3_tests/lib/python3.5/site-packages/requests/api.py:44: in request
    return session.request(method=method, url=url, **kwargs)
../../../.venv/ansible3_tests/lib/python3.5/site-packages/requests/sessions.py:335: in request
    resp = self.send(prep, **send_kwargs)
../../../.venv/ansible3_tests/lib/python3.5/site-packages/requests/sessions.py:438: in send
    r = adapter.send(request, **kwargs)
../../../.venv/ansible3_tests/lib/python3.5/site-packages/requests/adapters.py:292: in send
    timeout=timeout
../../../.venv/ansible3_tests/lib/python3.5/site-packages/requests/packages/urllib3/connectionpool.py:423: in urlopen
    conn = self._get_conn(timeout=pool_timeout)
../../../.venv/ansible3_tests/lib/python3.5/site-packages/requests/packages/urllib3/connectionpool.py:238: in _get_conn
    return conn or self._new_conn()
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
    def _new_conn(self):
[...]
        connection = VerifiedHTTPSConnection(host=self.host,
                                             port=self.port,
>                                            strict=self.strict)
E       TypeError: __init__() got an unexpected keyword argument 'strict'
```